src/daemon/daemon_config.cpp

```diff
@@ -114,7 +118,12 @@ std::unique_ptr<const mp::DaemonConfig> mp::DaemonConfigBuilder::build()
     if (factory == nullptr)
         factory = platform::vm_backend(data_directory);
     if (update_prompt == nullptr)
-        update_prompt = platform::make_update_prompt();
+    {
+        if (mp::Settings::instance().get_as<bool>(mp::update_check_key))
+            update_prompt = std::make_unique<DefaultUpdatePrompt>();
```

Comment:
As you suggested - the default when in snap would be `false`, but if someone enabled it, https://multipass.run does link to https://snapcraft.io/multipass, so don't see why not.

Reply:
I just thought it could be confusing. From the perspective of a snap user who finds the setting, they may wonder if they're missing out by having it disabled &mdash; "aren't updates taken care of by snaps? this must have some other capability if they added it". They may fail to realize it is only useful in other scenarios.